client/src/app/pages/reports/components/identified-risks-table/identified-risks-table.tsx

```diff
@@ -1,161 +1,197 @@
-      transforms: [cellWidth(35)],
-      cellTransforms: [breakWord],
-      cellFormatters: [],
-    },
-    {
-      title: t("terms.application(s)"),
-      transforms: [cellWidth(15)],
-      cellTransforms: [breakWord],
-      cellFormatters: [],
-    },
-  ];
-  const filterCategories: FilterCategory<
-    ITableRowData,
-    "category" | "question" | "answer" | "applications"
-  >[] = [
-    {
-      key: "category",
-      title: t("terms.category"),
-      type: FilterType.search,
-      placeholderText:
-        t("actions.filterBy", {
-          what: t("terms.category").toLowerCase(),
-        }) + "...",
-      getItemValue: (item) => {
-        return item?.category || "";
-      },
-    },
-    {
-      key: "question",
-      title: t("terms.question"),
-      type: FilterType.search,
-      placeholderText:
-        t("actions.filterBy", {
-          what: t("terms.question").toLowerCase(),
-        }) + "...",
-      getItemValue: (item) => {
-        return item?.question || "";
-      },
-    },
-    {
-      key: "answer",
-      title: t("terms.answer"),
-      type: FilterType.search,
-      placeholderText:
-        t("actions.filterBy", {
-          what: t("terms.answer").toLowerCase(),
-        }) + "...",
-      getItemValue: (item) => {
-        return item?.answer || "";
-      },
-    },
-    {
-      key: "applications",
-      title: t("terms.name"),
-      type: FilterType.search,
-      placeholderText:
-        t("actions.filterBy", {
-          what: t("terms.name").toLowerCase(),
-        }) + "...",
-      getItemValue: (item) => {
-        const applicationNames = item?.applications.map((app) => app.name);
-        return applicationNames?.join("") || "";
-      },
-    },
-  ];
+  interface ITableRowData {
+    assessmentName: string;
+    questionId: string;
+    section: string;
+    question: string;
+    answer: string;
+    applications: Ref[];
+  }
 
-  const { filterValues, setFilterValues, filteredItems } = useLegacyFilterState(
-    tableData || [],
-    filterCategories
-  );
+  const tableData: ITableRowData[] = [];
 
-  const { currentPageItems, paginationProps } = useLegacyPaginationState(
-    filteredItems,
-    10
-  );
+  // ...
 
-  const rows: IRow[] = [];
-  currentPageItems.forEach((item) => {
-    rows.push({
-      cells: [
-        {
-          title: item.category,
-        },
-        {
-          title: item.question,
-        },
-        {
-          title: item.answer,
-        },
-        {
-          title: item.applications.map((f) => f.name).join(", "),
-        },
-      ],
+  assessments.forEach((assessment) => {
+    assessment.sections.forEach((section) => {
+      section.questions.forEach((question) => {
+        question.answers.forEach((answer) => {
+          if (answer.selected) {
+            const existingItemIndex = tableData.findIndex(
+              (item) =>
+                item.question === question.text &&
+                item.answer === answer.text &&
```

Comment:
So this will have the same problem of if an assessment has repeated questions / answers right?

Maybe building up a key based on the order fields would be better?

```js
itemId = [
  assessment.id,
  section.order,
  question.order,
  answer.order,
].join(".");
```